### PR TITLE
chore: conform R-setup.md to reality

### PR DESCRIPTION
**Pull request by niehaus59**
## Related Issues/PRs

#1943
https://github.com/microsoft/SynapseML/issues/1943

## What changes are proposed in this pull request?

_Briefly describe the changes included in this Pull Request._

Updated docs in R-setup.md to reflect the fact we don't have an uber package for R

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [x] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [x] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/reference/R-setup.md](https://github.com/microsoft/SynapseML/blob/6c34d200d11c7621194ef936e1ccb163115c5bbc/website%2Fdocs%2Freference%2FR-setup.md) | 79 | 100 | 70 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/999e2f7c-cac2-4b93-9798-f2ad82a98592) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/reference/R-setup.md](https://github.com/microsoft/SynapseML/blob/6e2c5f02ed750361bdb777008b20af5033ec7588/website%2Fdocs%2Freference%2FR-setup.md) | 79 | 100 | 70 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9be97617-18a0-4ac2-9f8b-235e64a67b4a) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/reference/R-setup.md](https://github.com/microsoft/SynapseML/blob/9cca7efd974ea25bfbb5b2b08fb12e49806a6b4a/website%2Fdocs%2Freference%2FR-setup.md) | 94 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/528d6e47-dabe-4975-96a7-200ad1c159f4) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/reference/R-setup.md](https://github.com/microsoft/SynapseML/blob/66c9c9ca2d298922212ff675b1d9a1d386c971f9/website%2Fdocs%2Freference%2FR-setup.md) | 94 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5a092f8b-40b5-4e72-a588-ffd0ced64562) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/reference/R-setup.md](https://github.com/microsoft/SynapseML/blob/6b0d1b47dc549791fa1a88f59d8438acfac829ba/website%2Fdocs%2Freference%2FR-setup.md) | 78 | 100 | 53 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0cde246-ec53-44c2-b38a-dae94a040f5e) | [:white_check_mark:](# "Successfully processed") |
| [website/docs/reference/developer-readme.md](https://github.com/microsoft/SynapseML/blob/6b0d1b47dc549791fa1a88f59d8438acfac829ba/website%2Fdocs%2Freference%2Fdeveloper-readme.md) | 84 | 100 | 65 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/877d35f1-e962-4ad3-8774-4d8ffb21ab63) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: website/docs/reference/developer-readme.md [hunk 1]
```diff
@@ -10,6 +10,32 @@ description: SynapseML Development Setup
...
+    - On Linux, add the following to your .bashrc:
+    <Tabs groupId="operating-systems">
+        <TabItem value="linux" label="Mac/Linux">
+
+            export SPARK_HOME=~/bin/spark-3.2.0-bin-hadoop3.2/
+            export PATH="$SPARK_HOME/bin:$PATH"
+
+        </TabItem>
+    </Tabs>
+
```
Comment (mhamilton723) [inline website/docs/reference/developer-readme.md:38]: This step isnt needed sbt sets this up for you

File: website/docs/reference/developer-readme.md [hunk 2]
```diff
@@ -35,6 +61,8 @@ description: SynapseML Development Setup
     `horovod` requirement in the environment.yml file, because horovod installation only
     supports Linux or macOS. Horovod is used only for namespace `synapse.ml.dl`.
     :::
+1. On Windows, install WinUtils
+    - Download [WinUtils.exe](https://github.com/steveloughran/winutils/blob/master/hadoop-3.0.0/bin/winutils.exe) and copy it into the `bin` directory of your Spark installation, e.g. C:\Users\user\AppData\Local\Spark\spark-3.3.2-bin-hadoop3\bin
```
Comment (mhamilton723) [inline website/docs/reference/developer-readme.md:65]: Steps should be
Download Winutils
Place it in C:\Program Files\Hadoop\bin creating folders if needed
Edit the environment variables to set HADOOP_HOME to C:\Program Files\Hadoop
Edit the path to contain %HADOOP_HOME%\bin

**Review by mhamilton723 — CHANGES_REQUESTED** [review]
Amazing

**Review by ppruthi — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/reference/R-setup.md](https://github.com/microsoft/SynapseML/blob/af2e3d01deb502db4d6b4c3029110f10bb6b6d5f/website%2Fdocs%2Freference%2FR-setup.md) | 78 | 100 | 53 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/586bcec0-379e-43b2-b768-f62d1e25b342) | [:white_check_mark:](# "Successfully processed") |
| [website/docs/reference/developer-readme.md](https://github.com/microsoft/SynapseML/blob/af2e3d01deb502db4d6b4c3029110f10bb6b6d5f/website%2Fdocs%2Freference%2Fdeveloper-readme.md) | 85 | 100 | 62 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a87c086c-d65b-4a41-a9a1-2fc2c62ef990) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**